### PR TITLE
Create quickstart commands doc

### LINKED ISSUE #169: [DOC] - Quickstart/Commands page for advanced users
### Preliminary Checks

- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/nebari-dev/nebari/discussions

### Summary

This page is intended to be a cheat sheet with all the important commands a regualr/savvy user will need to quickly get Nebari up and running.

Inspiration:
- https://cli.github.com/manual/index
- https://spacy.io/usage#quickstart
- https://www.gatsbyjs.com/docs/quick-start/ (best)

This page can live in the Getting started section.

### Steps to Resolve this Issue

n/a

### PR DESCRIPTION
**Pull request by pavithraes**
## Reference Issues or PRs

Closes #169 

## What does this implement/fix?

_Put an `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features to not work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [x] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [x] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [x] This content adheres to the Nebari style guides.

#### Non-text content

- [x] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If the content were to be read as plain text, it still makes sense, and no information is missing.

### Any other comments?

I've kept this quite minimal, and the commands are taken from other docs pages. I'd love all thoughts and feedback. :)

### CODE REVIEW

File: docs/docs/getting-started/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
+---
+id: quickstart
+title: Quickstart
+description: A cheat-sheet of Neabri commands for returning users.
+---
+
+This quickstart is a Nebari commands reference for experienced and returning users.
+If you're new to Nebari, start at [Installing Nebari](installing-nebari.md).
+
```
Comment (pavithraes) [inline docs/docs/getting-started/quickstart.md:9]: We can add the Nebari CLI diagram here, when it's ready.

Comment (pavithraes) [reply docs/docs/getting-started/quickstart.md:9]: WIP here: https://lucid.app/lucidchart/ebfd0c6f-34e4-4817-8d28-79b448fe7cd8/edit?viewport_loc=35%2C147%2C1579%2C916%2C0_0&invitationId=inv_e2e6a631-c4cc-4b6c-9a26-4f900790f4fb

Comment (pavithraes) [reply docs/docs/getting-started/quickstart.md:9]: WIP here: https://lucid.app/lucidchart/ebfd0c6f-34e4-4817-8d28-79b448fe7cd8/edit?viewport_loc=75%2C149%2C1579%2C916%2C0_0&invitationId=inv_e2e6a631-c4cc-4b6c-9a26-4f900790f4fb

**Comment by pavithraes** [comment]
TODO: Add `validate` and `render`

File: docs/docs/getting-started/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
+---
+id: quickstart
+title: Quickstart
+description: A cheat sheet of Neabri commands for returning users.
+---
+
+This quickstart is a Nebari commands reference for experienced and returning users.
```
[the remark this answers is not in the document]
Comment (kcpevey) [reply docs/docs/getting-started/quickstart.md:7]: Something about this bothers me. I go to quickstart when I just want to jump into a project without the fluff. I think this document does that... but when you tell me its only for experienced and returning users, it confuses me. A doc for "experienced and returning users" doesn't belong in "Getting Started"

I think maybe it should just be reworded to not say "for experienced and returning users" but definitely include the next line that points newbies to "installing". What do you think?

[the remark this answers is not in the document]
Comment (pavithraes) [reply docs/docs/getting-started/quickstart.md:7]: Thanks for the review!

> I think maybe it should just be reworded to not say "for experienced and returning users" but definitely include the next line that points newbies to "installing". What do you think?

I think this is a good idea, thanks! I'll make the update. :)

**Review by kcpevey — APPROVED** [review]

**Pull request merged**